homeassistant/components/min_max/sensor.py

```diff
@@ -1,4 +1,4 @@
-"""Support for displaying the minimal and the maximal value."""
+"""Support for displaying minimal, maximal, mean or median values."""
```

Comment:
This confusing situation is what happens if you end up naming an entity incorrectly 🤦

Reply:
Agreed! However, I did not create the component in the first place, just enhancing it. I reckon it will be little value in changing name and thus creating breaking changes.

Reply:
I agree. Just silly :)